target/maven-archiver/pom.properties

```diff
@@ -1,5 +1,5 @@
 #Generated by Maven
-#Thu Jun 06 16:49:34 UTC 2024
+#Fri Jun 07 16:38:54 UTC 2024
```

Comment:
The fact that this file includes a timestamp means that it probably shouldn't be version-controlled.